pkg/pgmodel/ingestor/copier.go

```diff
@@ -33,7 +33,7 @@ var getBatchMutex = &sync.Mutex{}
 
 // Handles actual insertion into the DB.
 // We have one of these per connection reserved for insertion.
-func runCopier(conn pgxconn.PgxConn, in chan copyRequest) {
+func runCopier(conn pgxconn.PgxConn, decompressChunks bool, in chan copyRequest) {
```

Comment:
hmm. Why don't we pass down ingestor.Cfg instead and add this as a field to that struct?

Reply:
We already have decompressChunks in the struct. I assume you mean to pass cfg just on this level.